### DISCUSSION TITLE
Are there any actual details of this project?

### DISCUSSION THREAD

**Discussion by cybercrypt13** [universalweb/Network#8] [General]
I visited the sentivate site but still confused.
Thanks,

Glenn

**Comment by Tech-no-logic-cat** [comment]
Hi, sorry delay.
Sentivate is a small small company, self sustaining with customs works, no IDo or other finance ways, that is rebuilding the web .  The code is open source and I tested it succesfully on locale. Actually it still in the early stage and things are going on with realistic timing and  updates. They choose the hard way to develop this project. Follow @tommarchi on twitter for updated info. I also suggest you to make the time to read the whitepaper.
Regards

**Comment by tomekmarchi** [comment]
Check out the FAQ and read the docs in the repo plus whitepaper. They have all be dumbed down dramatically and are sourced. If you can't understand it then I doubt you'll be able to even with the technical documentation. If you have a dev background you should be able to easily manage. Else I would steer away from trying to help as that's what we are looking for in terms of assistance. None the less we don't let anyone besides specific Github users post to this repo especially regarding code. Language and text updates are welcome but again if you aren't a dev or capable enough of reading the commands in the repo or don't have a Nodejs background I'd sit this one out for the time being. Most topics we discuss go right over most devs heads. Very few venture where we have gone tech wise. I can only think of a few other initiatives that are just foundations of knowledge that can help you understand everything. The founding of the WWW, MinimaLT, Websockets, and QUIC will shed great light on the problems we are addressing and the technology we are building/built. Understanding those technologies will help you understand what is being built and how to go through the code. Code wise we follow an extremely detailed style and linting guide that I doubt will be found elsewhere. We are very strict about organization in the code base thus it should be easy to decipher and go through.

If you are still stuck in Ruby, PHP, Python, and the like worlds this is indeed not for you to explore unless you are open to reading a lot of bleeding edge web topics and can handle yourself code wise. Personally I can jump into any code base off rip and breakdown or reverse engineer even obfuscated code. If you are coming from the PHP MYSQL Web world I would suggest first starting looking into modern Web development trends like NodeJS and reactive components that would be a good place to start if new.

To summarize for you. It's a Web replacement which incorporates blockchain tech. A new Web with a new protocol, DNS, Browser, cryptography, and even how apps are built. The huge differences between the two can be gleamed from the Site/FAQ pages and most specifically the White paper. None the less based on pure probability I'd assume you aren't in any of those topics which is totally fine to each their own thus it's better to jump and try to swim rather than a slower approach. 

Sidenote alcohol is poison don't drink it and smoking is bad for your lungs.
